tests/accounts/test_integration.py

```diff
@@ -52,11 +52,32 @@ def test_existing_account_is_returned(self):
                          self.account2['accountInfo']['accountNumber'])
 
 
+class TestAccountCreationWithUserTokenPassed(AccountTestCase):
+    def setUp(self):
+        service = SitewitService()
+        site_id = uuid.uuid4()
+
+        response = service.create_account(
+            site_id, self.url, self.user_name, self.user_email,
+            self.currency, self.country_code)
+
+        self.user_token = response['userInfo']['token']
+
+        self.response = service.create_account(
+            uuid.uuid4(), 'https://foo.bar', self.user_name,
+            '{}@yola.com'.format(uuid.uuid4()),
+            self.currency, self.country_code, self.user_token)
+
+    def test_creates_new_account_for_the_given_user(self):
+        self.assertIsNotNone(self.user_token)
+        self.assertEqual(self.user_token, self.response['userInfo']['token'])
```

Comment:
It looks like I confused everyone again... :(   let me try to explain,  both new user and new account for that user are created during first call to SiteWit (there is no separate call to create user, and user != account on their side). For all subsequent calls (when we pass user_token), new user shouldn't be created - only account.